TestPlaybooks/playbook-DomainTools_Iris_Test.yml

```diff
@@ -0,0 +1,1177 @@
+id: DomainTools Iris - Test
```

Comment:
The test looks much better now. Good job!
Still need to check for some outputs. Choose some outputs that you expect to see from the call for `Indicator Pivoting - DomainTools Iris` to make sure that the playbook outputs what its supposed to. No need to test ALL the outputs though.
Also, is it incidental that there is nothing in `NameServerIPAddress` and others? Or are you inputting the wrong data?
![image](https://user-images.githubusercontent.com/43602124/67638831-ae80e380-f8f1-11e9-9d3e-2b1098dd3419.png)

Notice that 2 tasks ended up not running:
![image](https://user-images.githubusercontent.com/43602124/67638848-e851ea00-f8f1-11e9-8f4c-94a78a35f6ff.png)

Reply:
I am testing fewer things in the pivot playbook to save time. Hopefully as this is just a demonstration of what the playbook can do this is alright. Otherwise the playbook takes awhile to run even in a test environment.

Reply:
What you did is exactly what was needed :)